packages/database/src/migrations/20201006211507-BuildAncestorDescendantRelationCache-modifies-data.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
```

Comment:
Now that full build and cache is only 40 seconds, we can do it pretty happily in a migration! 🎉